dangerzone/isolation_provider/container.py

```diff
@@ -56,7 +56,12 @@ def get_runtime_version() -> Tuple[int, int]:
         """
         # Get the Docker/Podman version, using a Go template.
         runtime = Container.get_runtime_name()
-        cmd = [runtime, "version", "-f", "{{.Client.Version}}"]
+        if runtime == "podman":
+            query = "{{.Client.Version}}"
+        else:
+            query = "{{.Server.Version}}"
```

Comment:
At least on my machine, `podman version -f "{{.Client.Version}}"` returns the same than `podman version -f "{{.Server.Version}}"`.

I might not understand why we check for the "Server" version for Docker and the "Client" version for podman.

Reply:
I'm kind of hesitant to use `{{.Server.Version}}` for Podman, since it's not part of the JSON output:

```
$ podman version -f json | python3 -m json.tool
{
    "Client": {
        "APIVersion": "5.1.1",
        "Version": "5.1.1",
        "GoVersion": "go1.22.3",
        "GitCommit": "bda6eb03dcbcf12a5b7ae004c1240e38dd056d24-dirty",
        "BuiltTime": "Wed Jun  5 01:12:10 2024",
        "Built": 1717539130,
        "OsArch": "linux/amd64",
        "Os": "linux"
    }
}
```

I guess they internally satisfy this version check to match Docker's but I'm not sure if we can rely on it forever.